[API] change new-api prefix to api (#12257)
Sylius/Sylius: UPGRADE-1.9.md

   +           xml_serialization:
   ```

1. Unified API parameters have been changed to

>     ```diff   

Reviewer: Missing which file should I modify.

        parameters:
    -       sylius.security.api_regex: "^/api"
    -       sylius.security.shop_regex: "^/(?!%sylius_admin.path_name%|new-api|api/.*|api$|media/.*)[^/]++"
    -       sylius.security.new_api_route: "/new-api"
    +       sylius.security.api_regex: "^/api/v1"
    +       sylius.security.shop_regex: "^/(?!%sylius_admin.path_name%|api/.*|api$|media/.*)[^/]++"
    +       sylius.security.new_api_route: "/api/v2"
    ```

> 1. fos_rest rules have been changed to

Reviewer: ```suggestion
1. `fos_rest` rules have been changed to
```

>     ```diff   

Reviewer: Missing which file should I modify.

1. We've replaced deprecated Symfony Translator API with the new one.
   
    Replace `Symfony\Component\Translation\TranslatorInterface` with `Symfony\Contracts\Translation\TranslatorInterface` in your codebase.

> 1. We've changed `/new-api` prefix to `/api/v2`. If you're using API, replace api request url to `/api/v2`.

Reviewer 2: ```suggestion
1. `/new-api` prefix has been changed to `/api/v2`. Please adjust your routes accordingly.
```

>    Old api is hardcoded to `/api/v1`.

Reviewer 2: ```suggestion
   Admin Api was hardcoded to `/api/v1`(instead of /api/v{version}).
```

Reviewer 3: If I see correctly, in Admin API there is still `/api/v{version}`

>    Admin api is hardcoded to `/api/v1` (instead of `/api/v{version}`).

Reviewer: ```suggestion
   Admin API is hardcoded to `/api/v1` instead of `/api/v{version}`.
```

Reviewer: We could also add something like "Admin API will assume v1 for every request."
